libs/model/src/community/GetStakeHistoricalPrice.query.ts

```diff
@@ -16,11 +16,11 @@ export const GetStakeHistoricalPrice: Query<
         SELECT s.community_id, (s.stake_price / s.stake_amount::REAL)::TEXT as old_price
         FROM "StakeTransactions" s
         INNER JOIN (
-            SELECT community_id, MAX(timestamp) AS latest_timestamp
+            SELECT community_id, MIN(timestamp) AS latest_timestamp
```

Comment:
MIN would be earliest timestamp, yet it's labeled as "latest". That's a bit confusing.

Reply:
Good catch